doc/Command_Index_User.tex

```diff
@@ -155,6 +154,20 @@ \subsubsection{Including scores}
   \gabcsnippet[0]{(c3) Al(eg~)le(gv.fhg)lu(efe___)ia(e.) (::)}
 \end{latexcode}
 
+\macroname{\textbackslash greforcehyphen}{}{gregoriotex.tex}
+Tell Gregorio\TeX\ to force the appearance of hyphens between all
+syllables in polysyllabic words.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
+
+Note: This command only affects the hyphens within a score.  Hyphens
```

Comment:
I think this not is unnecessary, as the opposite would just be a bug, not a reasonable design choice...